mappings/net/minecraft/world/biome/Decorator.mapping

```diff
@@ -0,0 +1,17 @@
+CLASS net/minecraft/class_725 net/minecraft/world/biome/Decorator
+	FIELD field_2345 deadbushCount I
+	FIELD field_2353 generateSprings Z
+	FIELD field_2354 decorator Lnet/minecraft/class_26;
+	FIELD field_2355 rand Ljava/util/Random;
```

Comment:
ive been using random instead of rand for all of the random fields in these lands :P